cmake/Findquatlib.cmake

```diff
@@ -65,19 +65,35 @@ else()
 		/usr/local)
 
 	# Look for the library.
-	find_library(QUATLIB_LIBRARY
+	find_library(QUATLIB_LIBRARY_RELEASE
 		NAMES
 		quat.lib
 		libquat.a
-		HINTS
+		PATH_SUFFIXES
+		ReleaseAcademicEdition
+		Release
+		${_libsuffixes}
+		PATHS
```

Comment:
The previous code distinguished the supplied `QUATLIB_ROOT_DIR` by putting it under `HINTS` rather than `PATHS` - this code changes it to just be the first in `PATHS`. I suspect this is probably actually the correct convention, but wanted to make sure that you had made this change intentionally for that reason (improving convention) and not confusion or issues regarding `PATH_SUFFIXES`. `PATH_SUFFIXES` are applied equally to  `HINTS`, the built-in search paths, and `PATHS`, in that order, if I remember correctly